qa/rpc-tests/README.md

```diff
@@ -36,7 +36,7 @@ If you set the environment variable `PYTHON_DEBUG=1` you will get some debug out
 A 200-block -regtest blockchain and wallets for four nodes
 is created the first time a regression test is run and
 is stored in the cache/ directory. Each node has 25 mature
-blocks (25*50=1250 BTC) in its wallet.
+blocks (25*50=1250 ZEC) in its wallet.
```

Comment:
The per-block miner reward for regtest is either 10 or 12.5 ZEC, not 50 ZEC. (I'm not sure which, it depends whether you need to pay the Founders' Reward on regtest.)

Reply:
It's also imprecise to say that there are "blocks" in the wallet; change it to "Each node has the miner subsidy from 25 mature blocks ...".